cores/esp8266/FSImpl.h

```diff
@@ -115,6 +115,7 @@ class FSImpl {
     virtual bool rmdir(const char* path) = 0;
     virtual bool gc() { return true; } // May not be implemented in all file systems.
     virtual bool check() { return true; } // May not be implemented in all file systems.
+    virtual time_t creationTime() { return -1; } // May not be implemented in all file systems.
```

Comment:
Can we make this `getCreationTime` like the existing `File::getCreationTime`  call?

Reply:
Sure, that makes sense.